secretariat/management/commands/import-from-outline.py

```diff
@@ -86,4 +97,132 @@ def create_new_django_user(self, outline_user):
+                                f"{prompt_already_existing}, sans UUID précisé. UUID copié depuis Outline."
+                            )
+                        )
+                        django_orga.outline_group_uuid = outline_group["id"]
+                        django_orga.save()
+                    else:
+                        self.stdout.write(
+                            self.style.ERROR(
+                                f"{prompt_already_existing}. Les UUID ne correspondent pas (django: '{django_orga.outline_group_uuid}', outline: '{outline_group['id']}')."
+                            )
+                        )
+                        count_errors += 1
+            else:
+                try:
+                    django_orga = self.create_new_django_orga(outline_group)
+                    count_new_groups += 1
+                except IntegrityError:
+                    count_errors += 1
+
+            # then lists group members on django and outline
+
+            # we then prepare a set listing every member of the group in Django. We'll remove outline members.
+            # The remaining list should be empty. If not, users might have been removed directly in outline of here
+            # django users were not properly synced to outline
+            django_users_unaccounted_for = set(
+                value[0] for value in django_orga.members.values_list("username")
+            )
+
+            # and creates memberships for all of them
+            outline_group_members = client.list_group_users(outline_group["id"])
+            count_added_members = 0
+            for member in outline_group_members:
+                django_user = User.objects.get(outline_uuid=member["id"])
+
+                try:
+                    membership, is_created = Membership.objects.get_or_create(
+                        user=django_user,
+                        organisation=django_orga,
+                        role="admin" if member["isAdmin"] else "member",
+                    )
+                    if is_created:
+                        count_added_members += 1
+                    else:
+                        django_users_unaccounted_for.remove(django_user.username)
+                except Exception as exception:
+                    print("Exception : ", exception)
+
+            if count_added_members != 0:
+                print(
+                    f"{count_added_members} membres ajouté.es au groupe {django_orga.name}."
+                )
+
+        # Every member should be accounted for. Warn user otherwise
+        users_with_unexpected_auth = set(django_users_unaccounted_for)
+        print(len(users_with_unexpected_auth), "souci(s) de permissions après import.")
+        if len(users_with_unexpected_auth) != 0:
+            print("Veuillez vérifier/synchroniser les groupes des membres suivants :")
+            for extra_django_member in users_with_unexpected_auth:
+                print("   - ", extra_django_member)
```

Comment:
Sauf erreur de ma part, `django_users_unaccounted_for est déjà un set donc tu peux simplifier et réutiliser la variable existante : 

```suggestion
        # Every member should be accounted for. Warn user otherwise
        print(len(django_users_unaccounted_for), "souci(s) de permissions après import.")
        if len(django_users_unaccounted_for) != 0:
            print("Veuillez vérifier/synchroniser les groupes des membres suivants :")
            for extra_django_member in django_users_unaccounted_for:
                print("   - ", extra_django_member)
```

Reply:
Justement, les deux variables sont des sets. C'est pour ça que je suis intriguée.